app/controllers/superpowers_controller.rb

```diff
@@ -52,4 +53,9 @@ def superpower_params
     params.require(:superpower).permit(:name, :description)
   end
 
+  def authorize_user
```

Comment:
It might be better to render a status code of 401 unauthorized or redirect the user with a flash message that tells them they tried to access something they aren't authorized to see.

Reply:
I would use a similar approach to how you authorize the deletion of users